package.json

```diff
@@ -29,9 +29,9 @@
     }
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/topo": "^5.0.0",
-    "@vrbo/determination": "^6.0.1",
+    "@hapi/hoek": "^11.0.2",
+    "@hapi/topo": "^6.0.0",
```

Comment:
Upgrading both of these will cause conflicts in other packages that consume this one.  

Minimally, they should be the lowest, newest, version possible (`^11.0.0` vs. `^11.0.2`); but I'm tempted to say we should use a range expansion (`^9.0.4 || ^10.0.0 || ^11.0.0`) on these until `hapi` upgrades can be rolled out across the product line.